apps/connect/source/ftrack_connect/utils/login.py

```diff
@@ -8,47 +8,45 @@
 import logging
 
 
-logger = logging.getLogger('ftrack_connect.ui.config')
+logger = logging.getLogger('ftrack_connect.utils.loging')
 
 
-def get_config_file_path():
+def get_login_file_path():
     '''Return config file path.'''
-    config_file = os.path.join(
-        platformdirs.user_data_dir('ftrack-connect', 'ftrack'), 'config.json'
+    login_file = os.path.join(
+        platformdirs.user_data_dir('ftrack-connect', 'ftrack'), 'login.json'
     )
 
-    return config_file
+    return login_file
 
 
-def read_json_config():
+def read_json_login():
     '''Return json config from disk storage.'''
-    config_file = get_config_file_path()
-    config = None
+    login_file = get_login_file_path()
+    login = None
 
-    if os.path.isfile(config_file):
-        logger.debug(u'Reading config from {0}'.format(config_file))
+    if os.path.isfile(login_file):
```

Comment:
It seems like it does not port over the existing config.json, I think we need to do this so all users do not need to login to Connect again.